doc/classes/LineEdit.xml

```diff
@@ -24,7 +24,7 @@
 			<return type="void">
 			</return>
 			<description>
-				Erases the [code]LineEdit[/code] text.
+				Erases the [LineEdit] text.
```

Comment:
This was done on purpose to avoid self-referencing hyperlinks. The docs guidelines specify that when referencing Self in the docs, it should use `[code]` instead of `[Self]`.

Reply:
Could we change this and move that logic to EditorHelp and makerst (to not generate an hyperlink if it's to the current class)? It's harder to parse the content of code tags.
I'm willing to do these changes and replace existing cases in the classref if approved.

Reply:
I'd be fine with that yeah. For EditorHelp we can likely keep the hyperlink as they don't really hurt, it's more for the HTML that we want to avoid overloading the file with irrelevant external links.

Reply:
Done. Replaced all the `[code]CurrentClassName[/code]` occurences with `[CurrentClassName]` and made makerst output them as code tags as it was doing.